src/crypto.rs

```diff
@@ -19,10 +24,9 @@ use crate::{
     Error, Result, AES_128_KEY_LEN, AES_256_KEY_LEN, AES_BLOCK_SIZE,
 };
 
-// Reads an X509 cert chain (provided by the tenant with the --cert command) and outputs
-// its public key.
-pub(crate) fn import_x509(input_key_path: &Path) -> Result<PKey<Public>> {
-    let contents = fs::read_to_string(input_key_path)?;
+// Read a X509 cert or cert chain and outputs the first certificate
```

Comment:
What do you mean "first" certificate? Is this the leaf?

Reply:
We error if there find more than one, so we expect exactly one leaf